inst/examples/data/translation_it.csv

```diff
@@ -0,0 +1,6 @@
+en,it
```

Comment:
do we need these? Instead of `en` the keys can be anything.
why not simply `key,translation`?

Reply:
We agreed to leave it like this, but I added a function checking if key is consistent among all csv files.